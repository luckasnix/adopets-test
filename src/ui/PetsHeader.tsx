import React from 'react'
import styles from './PetsHeader.module.css'
import { ReactComponent as AdopetsLogo } from '../../assets/images/adopets-logo.svg'

const PetsHeader: React.FunctionComponent = () => {
  return (
    <header className={styles.header}>
      <div className={styles.logo}>
        <AdopetsLogo/>
      </div>
    </header>
  )
}

export default PetsHeader