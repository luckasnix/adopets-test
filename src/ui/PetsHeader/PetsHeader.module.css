.header {
  width: 100%;
  height: 80px;
  display: grid;
  justify-content: center;
  align-content: center;
  background-color: #f6f9fa;
}
.logo {
  width: 200px;
  height: auto;
}
.logo svg {
  width: 100%;
  height: auto;
}