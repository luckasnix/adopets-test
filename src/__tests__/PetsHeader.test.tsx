import React from 'react'
import PetsHeader from '../ui/PetsHeader/PetsHeader'
import renderer from 'react-test-renderer'

it(
  'renders correctly',
  () => {
    const tree = renderer.create(<PetsHeader/>).toJSON()
    expect(tree).toMatchSnapshot()
  }
)