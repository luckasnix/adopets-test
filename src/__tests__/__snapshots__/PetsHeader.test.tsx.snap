// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header
  className="header"
>
  <div
    className="logo"
  >
    <svg>
      adopets-logo.svg
    </svg>
  </div>
</header>
`;
